content-review/impl/vericite/pom.xml

```diff
@@ -42,5 +52,62 @@
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpmime</artifactId>
 		</dependency>
+                <dependency>
+                        <groupId>commons-io</groupId>
+                        <artifactId>commons-io</artifactId>
+                </dependency>
+		<!-- HTTP client: jersey-client -->
+                <dependency>
+                        <groupId>com.sun.jersey</groupId>
+                        <artifactId>jersey-client</artifactId>
+                        <version>${jersey-version}</version>
+                </dependency>
+                <dependency>
+                        <groupId>com.sun.jersey.contribs</groupId>
+                        <artifactId>jersey-multipart</artifactId>
+                        <version>${jersey-version}</version>
+                </dependency>
+                <dependency>
+                        <groupId>io.swagger</groupId>
+                        <artifactId>swagger-annotations</artifactId>
+                        <version>${swagger-annotations-version}</version>
+                </dependency> 
+                <!-- JSON processing: jackson -->
+                <dependency>
```

Comment:
Jackson is already in shared just need to add dependency with groupId and artifactId.